reactive/jenkins.py

```diff
@@ -270,50 +279,56 @@ def migrate_charm_data():
     configuration.migrate()
 
 
-@when('nrpe-external-master.available')
+@when("nrpe-external-master.available")
 def update_nrpe_config(nagios):
     unit_data = unitdata.kv()
-    nagios_hostname = unit_data.get('nagios.hostname', None)
-    nagios_host_context = unit_data.get('nagios.host_context', None)
+    nagios_hostname = unit_data.get("nagios.hostname", None)
+    nagios_host_context = unit_data.get("nagios.host_context", None)
 
     # require the nrpe-external-master relation to provide the host context
-    if in_relation_hook() and relation_id().\
-            startswith('nrpe-external-master:'):
+    if in_relation_hook() and relation_id().startswith("nrpe-external-master:"):
         rel = relation_get()
-        if 'nagios_host_context' in rel:
-            nagios_host_context = rel['nagios_host_context']
-            unit_data.set('nagios.host_context', nagios_host_context)
+        if "nagios_host_context" in rel:
+            nagios_host_context = rel["nagios_host_context"]
+            unit_data.set("nagios.host_context", nagios_host_context)
 
             # We have to strip the nagios host context from the nagios hostname
             # since the nagios.add_check will put it back again...
-            nagios_hostname = rel['nagios_hostname']
-            if nagios_hostname.startswith(nagios_host_context + '-'):
-                nagios_hostname = nagios_hostname[len(nagios_host_context +
-                                                      '-'):]
+            nagios_hostname = rel["nagios_hostname"]
+            if nagios_hostname.startswith(nagios_host_context + "-"):
+                nagios_hostname = nagios_hostname[len(nagios_host_context + "-") :]
 
-            unit_data.set('nagios.hostname', nagios_hostname)
+            unit_data.set("nagios.hostname", nagios_hostname)
 
     if not nagios_hostname or not nagios_host_context:
         return
     # The above boilerplate is needed until this issue is fixed:
     #
     # https://github.com/cmars/nrpe-external-master-interface/issues/6
 
-    status_set('maintenance', 'Updating Nagios configs')
+    status_set("maintenance", "Updating Nagios configs")
 
     creds = Credentials()
     check = [
-        '/usr/lib/nagios/plugins/check_http', '-H', 'localhost', '-p',
-        '8080', '-u', urlparse(Api().url).path, '-a',
+        "/usr/lib/nagios/plugins/check_http",
+        "-H",
+        "localhost",
+        "-p",
+        "8080",
+        "-u",
+        urlparse(Api().url).path,
+        "-a",
         "{}:{}".format(creds.username(), creds.token()),
```

Comment:
f-string?

Reply:
As above, need to support Xenial which is Python 3.5 which doesn't support f-strings